model/client.go

```diff
@@ -18,6 +18,8 @@ import (
 	l4g "github.com/alecthomas/log4go"
 )
 
+var UsedApiV3 bool
```

Comment:
Don't think this is thread safe.  You might want to use an atomic int instead.

Reply:
Didn't think it would really matter but I'll make it atomic anyways